### DISCUSSION TITLE
Patroni - ETCD - Postgres15

### DISCUSSION THREAD

**Discussion by ganamur1983** [community/community#140764] [Programming Help]
### Select Topic Area

Question

### Body

[ETCD.txt](https://github.com/user-attachments/files/17275141/ETCD.txt)
Hi All,
I am getting below status for patroni on DB1 and DB2 node. Please suggest how to fix.
I am herewith attaching patroni.yml , etcd services and etcd.yml
Environment:
Two DB servers (Postgres15. RHEL 9)
1 ETCD server
[root@db2 db2]# etcdctl version
etcdctl version: 3.5.15
API version: 3.5
[patroni-yml.txt](https://github.com/user-attachments/files/17275137/patroni-yml.txt)

**[root@db2 db2]# sudo systemctl status patroni**
● patroni.service - Patroni Orchestration
     Loaded: loaded (/etc/systemd/system/patroni.service; enabled; preset: disabled)
     Active: active (running) since Mon 2024-10-07 12:56:19 IST; 8min ago
   Main PID: 6054 (patroni)
      Tasks: 2 (limit: 34980)
     Memory: 27.1M
        CPU: 606ms
     CGroup: /system.slice/patroni.service
             └─6054 /usr/bin/python3 /usr/local/bin/patroni /etc/patroni.yml

Oct 07 13:04:56 db2 patroni[6054]: 2024-10-07 13:04:56,213 ERROR: Failed to get list of machines from http://192.168.0.113:2379/v2: MaxRe>
Oct 07 13:04:56 db2 patroni[6054]: 2024-10-07 13:04:56,214 INFO: waiting on etcd
Oct 07 13:05:01 db2 patroni[6054]: 2024-10-07 13:05:01,221 WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=0, status=>
Oct 07 13:05:01 db2 patroni[6054]: 2024-10-07 13:05:01,223 WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=0, status=>
Oct 07 13:05:01 db2 patroni[6054]: 2024-10-07 13:05:01,225 ERROR: Failed to get list of machines from http://192.168.0.113:2379/v2: MaxRe>
Oct 07 13:05:01 db2 patroni[6054]: 2024-10-07 13:05:01,225 INFO: waiting on etcd
Oct 07 13:05:06 db2 patroni[6054]: 2024-10-07 13:05:06,232 WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=0, status=>
Oct 07 13:05:06 db2 patroni[6054]: 2024-10-07 13:05:06,233 WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=0, status=>
Oct 07 13:05:06 db2 patroni[6054]: 2024-10-07 13:05:06,234 ERROR: Failed to get list of machines from http://192.168.0.113:2379/v2: MaxRe>
Oct 07 13:05:06 db2 patroni[6054]: 2024-10-07 13:05:06,235 INFO: waiting on etcd
lines 1-20/20 (END)

**[root@etcd default]# sudo systemctl status etcd**
● etcd.service - Etcd Server
     Loaded: loaded (/usr/lib/systemd/system/etcd.service; enabled; preset: disabled)
     Active: active (running) since Mon 2024-10-07 12:48:27 IST; 21min ago
   Main PID: 7747 (etcd)
      Tasks: 6 (limit: 22411)
     Memory: 9.7M
        CPU: 2.497s
     CGroup: /system.slice/etcd.service
             └─7747 /usr/bin/etcd

Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.325320+0530","logger":"raft","caller":>
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.325388+0530","logger":"raft","caller":>
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.325452+0530","logger":"raft","caller":>
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.335508+0530","caller":"etcdserver/serv>
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.336157+0530","caller":"embed/serve.go:>
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.336760+0530","caller":"etcdmain/main.g>
Oct 07 12:48:27 etcd systemd[1]: Started Etcd Server.
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.343726+0530","caller":"etcdmain/main.g>
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.344157+0530","caller":"v3rpc/health.go>
Oct 07 12:48:27 etcd bash[7747]: {"level":"info","ts":"2024-10-07T12:48:27.345033+0530","caller":"embed/serve.go:>
lines 1-20/20 (END)

**Comment by ganamur1983** [comment]
192.168.0.109 - db1
192.168.0.112 - db2
192.168.0.113 -etcd

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ganamur1983 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion. You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your questions entails-- updating the title and giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
